core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -351,13 +345,10 @@ private[spark] object UIUtils extends Logging {
             <div class="brand">
               <a href={prependBaseUri("/")} class="brand">
                 <img src={prependBaseUri("/static/snappydata/SnappyData-Logo-230X50.png")} />
-                <!-- <span class="version">{org.apache.spark.SPARK_VERSION}</span> -->
+                {getProductUINameNode}
                 {getProductVersionNode}
               </a>
             </div>
-            <p class="navbar-text pull-right">
-              <strong title={appName}>{shortAppName}</strong> application UI
-            </p>
```

Comment:
It seems like both simpleSparkPageWithTabs and simpleSparkPageWithTabs_2 are SnappyData specific changes. These both should be moved to SnappyDashboardPage instead of polluting UIUtils with SnappyData specific content.